modules/Core/pages/profile.php

```diff
@@ -108,6 +108,18 @@
                                 ]
                             );
 
+                            EventHandler::executeEvent('userNewProfilePost', [
```

Comment:
Currently you use user language, you need to use default language

use `$default_language = new Language('core', DEFAULT_LANGUAGE);`

Reply:
Sorry I don't understand what you mean :c

Reply:
Resolved

Reply:
Thanks - I understand what partydragen meant now